android/res/values/strings.xml

```diff
@@ -409,6 +409,7 @@
 	<string name="editor_hours_closed">Non-Business Hours</string>
 	<string name="editor_example_values">Example Values</string>
 	<string name="editor_correct_mistake">Correct mistake</string>
+	<string name="editor_invalid_format">Format not valid!</string>
```

Comment:
This change should not be done in this commit. It should be autogenerated by the script.